dhall-nix/dhall-nix.cabal

```
@@ -30,7 +30,7 @@ Library
         base                      >= 4.11.0.0 && < 5   ,
         containers                               < 0.7 ,
         data-fix                                 < 0.4 ,
-        dhall                     >= 1.35     && < 1.36,
+        dhall                     >= 1.35     && < 1.37,
```

`Dhall.Nix` references `TextReplace`

```suggestion
        dhall                     >= 1.36     && < 1.37,
```